ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/IntegratedWindowsAuthentication/README.md

```diff
@@ -16,7 +16,7 @@ IWA, which is built into Microsoft Internet Information Server (IIS), works well
 
 * Search Secure
 
-  Enter the URL(e.g. https://myorgiwaportal.myorg.com/gis) to your IWA-secured portal in the text field at the top of the screen. Click the "Search Secure". You will be prompted for a username (including domain, such as username@DOMAIN or domain\username), and password. On Windows it will automatically use your current login credentials. If you authenticate successfully, portal item results will display in the combo box below. Select a web map item and click the "load web map" button to display it in the map view.
+  Enter the URL to your IWA-secured portal in the text field at the top of the screen. Click the "Search Secure". You will be prompted for a username (including domain, such as username@DOMAIN or domain\username), and password. On Windows it will automatically use your current login credentials. If you authenticate successfully, portal item results will display in the combo box below. Select a web map item and click the "load web map" button to display it in the map view.
```

Comment:
>  Click the "Search Secure"

extra "the". I think a single sentence would read better:

> Enter the URL to your IWA-secured portal in the text field at the top of the screen and click "Search Secure".

Reply:
> Select a web map item and click the ~~"load web map"~~ "Load Web Map" button

I would capitalize the button name to match the original text and to be consistent with "Search Secure" mentioned earlier